### PR TITLE
Moving contributing file and adding pull request template

### PR DESCRIPTION
**Pull request by cshinn**
## Description
This PR adds a a pull request template file to help our contributors make effective PRs and contribute patterns that meet our expectations

## Changes

- Moves CONTRIBUTING.md to the .github folder
- Adds PULL_REQUEST_TEMPLATE.md file

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+* Change 01
+* Change 02
+* Change 03
+
+## New Pattern Checklist (if relevant)
+
+- [ ] Pattern does not already exist
+- [ ] Pattern solves common problems
+- [ ] Pattern has community interest (You've addressed the PatternFly e-list)
```
Comment (andresgalante) [inline .github/PULL_REQUEST_TEMPLATE.md]: you mean the patternfly "mailing" list, right?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- [ ] All required components exist
```
Comment (andresgalante) [inline .github/PULL_REQUEST_TEMPLATE.md]: I understand what you are trying to say here. I wonder if this is too much.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- [ ] New files are in a folder titled with the name of the pattern
+- [ ] All images are in an "img" folder
```
Comment (andresgalante) [inline .github/PULL_REQUEST_TEMPLATE.md]: great point. can we append to this something about you've tested that the images have the correct path?

**Comment by andresgalante** [comment]
Thanks @cshinn, I've made a few comments

**Comment by andresgalante** [comment]
Thanks a lot @cshinn 

@LHinson I am ok to merge this 🚀

**Comment by LHinson** [comment]
bitmoji

**Pull request merged**